### ISSUE TITLE
containers-0.7 not accepted

### ISSUE THREAD

**Issue by ysangkok** [mbg/wai-saml2#61]
As you can see on [version history](https://gitlab.haskell.org/ghc/ghc/-/wikis/commentary/libraries/version-history), GHC 9.10 has containers-0.7. However , this version is excluded by wai-saml2.cabal. It would nice to accept this version. That might unblock using wai-saml2 on GHC 9.10.

**Comment by mbg** [comment]
Thanks for reporting this -- I am having a look to see what it takes to get this to compile with GHC 9.10

**Comment by ysangkok** [comment]
Note that it *can* also be tested with older versions of GHC. For example, this passes for me:

```
$ cabal test -w ghc-9.8.2 -c 'containers>=0.7' --allow-newer=wai-saml2:containers
Running 1 test suites...
Test suite parser: RUNNING...
Test suite parser: PASS
Test suite logged to:
/tmp/wai-saml2/dist-newstyle/build/x86_64-linux/ghc-9.8.2/wai-saml2-0.5/t/parser/test/wai-saml2-0.5-parser.log
1 of 1 test suites (1 of 1 test cases) passed.
```

**Comment by mbg** [comment]
I have published [v0.6 of this library](https://hackage.haskell.org/package/wai-saml2-0.6) which includes the increased bounds for `containers` as well as the dependencies on the `crypton-*` packages.

**Comment by ysangkok** [comment]
Excellent, thank you very much
